build/ProjectReunion-release.yml

```diff
@@ -0,0 +1,206 @@
+
+- job: SignBinariesAndPublishSymbols
+  dependsOn: Build
+  condition:
+    in(dependencies.Build.result, 'Succeeded', 'SucceededWithIssues', 'Skipped')
+  pool:
+    name: Package ES Lab E
+
+  steps:
+  - task: PkgESSetupBuild@10
+    displayName: 'XESSetupBuild'
+    inputs:
+      productName: projectreunion
+      branchVersion: false
+      nugetVer: true
+
+  - task: DownloadBuildArtifacts@0
+    condition:
+      eq(variables['useBuildOutputFromBuildId'],'')
+    inputs:
+      artifactName: drop
+      downloadPath: '$(Build.ArtifactStagingDirectory)'
+
+  - task: DownloadBuildArtifacts@0
+    condition:
+      ne(variables['useBuildOutputFromBuildId'],'')
+    inputs:
+      buildType: specific
+      buildVersionToDownload: specific
+      project: $(System.TeamProjectId)
+      pipeline: $(System.DefinitionId)
+      buildId: $(useBuildOutputFromBuildId)
+      artifactName: drop
+      downloadPath: '$(Build.ArtifactStagingDirectory)'
+
+  - task: CmdLine@1
+    displayName: 'Display build machine environment variables'
+    inputs:
+      filename: 'set'
+
+  - script: cmd /c dir /s /b $(Build.ArtifactStagingDirectory)
+    displayName: Dump artifact staging directory
+
+  - template: AzurePipelinesTemplates\ProjectReunion-PopulateBuildDateAndRevision-Steps.yml
+
+  - task: CopyFiles@2
+    displayName: 'Copy symbols to artifact staging directory'
+    condition: always()
+    inputs:
+      sourceFolder: $(Build.ArtifactStagingDirectory)\drop
+      contents: |
+        **\*.pdb
+      targetFolder: $(Build.ArtifactStagingDirectory)\symbols
+      
+# dump symbols artifact directory - this is normally turned off
+#  - script: |
+#      echo $(Build.ArtifactStagingDirectory)\symbols
+#      dir /s $(Build.ArtifactStagingDirectory)\symbols
+
+  - task: PublishSymbols@2
+    displayName: 'Publish symbols (without source indexing)'
+    inputs:
+      searchPattern: '$(Build.ArtifactStagingDirectory)\symbols\**\*.pdb'
+      symbolServerType: 'TeamServices'
+# this ADO task does not support indexing of github sources currently :-(      
+      indexSources: false 
+      detailedLog: true
+    # There is a bug which causes this task to fail if LIB includes an inaccessible path (even though it does not depend on it).
+    # To work around this issue, we just force LIB to be any dir that we know exists.
+    env:
+      LIB: $(Build.SourcesDirectory)
+
+  - task: PkgESCodeSign@10
+    displayName: CodeSign
+    env:
+      SYSTEM_ACCESSTOKEN: $(system.accesstoken)
+    inputs:
+      signConfigXml: '$(Build.SourcesDirectory)\build\SignConfig.xml'
+      inPathRoot: '$(Build.ArtifactStagingDirectory)\drop'
+      outPathRoot: '$(Build.ArtifactStagingDirectory)\drop'
+
+  # Re-publish signed artifacts to the drop.
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish artifact: drop'
+    inputs:
+      PathtoPublish: '$(Build.ArtifactStagingDirectory)\drop'
+      artifactName: 'drop'
+
+# Create Nuget Package
```

Comment:
>Nuget Package [](start = 9, length = 13)

Nuget package also needs signed.

Reply:
The nupkg has a .signature file that looks right/good in certmgr.exe.  CreateNugetPackage uses a special signconfig for signing the nuget package.

Reply:
Yes looks like createNugetPackage-job.ml is doing the correct Nuget signing with the packageES task